requirements.txt

```diff
@@ -1,6 +1,7 @@
 tensorflow~=2.3.0
 tfaip==1.0.1
 scikit-image>=0.15.0
+opencv-python
```

Comment:
I'd recommend `opencv-python-headless` which is slightly smaller and is easier to use in containerized environments without GUI libraries.